object.c

```diff
@@ -1672,7 +1672,8 @@ rb_class_inherited_p(VALUE mod, VALUE arg)
  *   mod < other   ->  true, false, or nil
  *
  * Returns true if <i>mod</i> is a subclass of <i>other</i>. Returns
- * <code>nil</code> if there's no relationship between the two.
+ * <code>false</code> if two modules are the same.
```

Comment:
It also returns `false` when `mod` is a superclass of `other`. For example:

```ruby
class Base; end
class Child < Base; end
puts Child < Base # => true
puts Base < Child # => false
puts Child < Child # => false
```

Reply:
@peterzhu2118 Thanks for the input. I have updated the PR to reflect the requested changes.